raise-lock-pr/action.yaml

```diff
@@ -4,13 +4,20 @@ inputs:
   token:
     description: 'Github auth token with permissions to raise a PR'
     required: true
+  lock-settings:
+    description: 'shell command to run after updating lock files. Can be used to cleanup locks'
+    default: 'false'
 runs:
   using: composite
   steps:
     - name: Update lockfiles
       uses: hypertrace/github-actions/gradle@main
       with:
-        args: resolveAndLockAll --write-locks
+        args: resolveAndLockAll --write-locks --refresh-dependencies
+    - name: Remove settings lock if needed
+      if: inputs.lock-settings == 'true'
+      shell: bash
+      run: rm settings-gradle.lockfile
```

Comment:
```suggestion
      run: rm settings-gradle.lockfile || true
```

Reply:
It should always be present if the repo us using locks in the first place, but doesn't hurt to make it robust.